scheduler/pkg/kafka/gateway/infer.go

```diff
@@ -241,7 +287,7 @@ func (kc *InferKafkaHandler) AddModel(modelName string) error {
 	kc.subscribedTopics[inputTopic] = true
 	err := kc.subscribeTopics()
 	if err != nil {
-		kc.logger.WithError(err).Warn("Failed to subscribe to topics")
+		kc.logger.WithError(err).Errorf("failed to subscribe to topics")
 		return nil
```

Comment:
we should investigate more on why all of these errors are being swallowed in `infer.go` instead of being returned